changelog/5.0.3_2024-05-02/bump-reva.md

```diff
@@ -0,0 +1,11 @@
+Bugfix: Update reva to v2.19.5
+
+We updated reva to v2.19.5
```

Comment:
Still reva with 2.19.5 in 5.0.3?
do yo plan a post fix to correct it to 2.19.6?

Reply:
good catch, pushing fix